package.json

```diff
@@ -1,97 +1,48 @@
-  },
-  "dependencies": {
-    "@aws-amplify/adapter-nextjs": "^1.2.23",
-    "@aws-amplify/backend": "^1.2.0",
-    "@aws-sdk/client-ses": "^3.637.0",
-    "date-fns": "^4.1.0",
-    "markdown-it": "^13.0.1",
-    "mimetext": "^3.0.24",
-    "next": "14.2.13",
-    "nhsuk-frontend": "^8.3.0",
-    "nhsuk-react-components": "^4.1.1",
-    "react": "^18",
-    "react-dom": "^18",
-    "winston": "^3.14.2",
-    "zod": "^3.23.8"
-  },
-  "devDependencies": {
-    "@aws-amplify/backend-cli": "^1.2.5",
-    "@testing-library/jest-dom": "^6.5.0",
-    "@testing-library/react": "^16.0.1",
-    "@testing-library/user-event": "^14.5.2",
-    "@types/jest": "^29.5.14",
-    "@types/markdown-it": "^13.0.1",
-    "@types/node": "^22.8.1",
-    "@types/react": "^18",
-    "@types/react-dom": "^18",
-    "@typescript-eslint/eslint-plugin": "^8.7.0",
-    "constructs": "^10.3.0",
-    "esbuild": "^0.24.0",
-    "eslint": "^8",
-    "eslint-config-airbnb": "^19.0.4",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-next": "14.2.6",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-import-resolver-typescript": "^3.6.3",
-    "eslint-plugin-html": "^8.1.1",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-jest": "^28.8.3",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-security": "^3.0.1",
-    "eslint-plugin-sonarjs": "^2.0.2",
-    "eslint-plugin-unicorn": "^55.0.0",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "jest-html-reporter": "^3.10.2",
-    "jest-mock-extended": "^3.0.7",
-    "mochawesome": "^7.1.3",
-    "nhs-notify-web-template-management-ui-tests": "file:tests/test-team",
-    "pa11y": "^8.0.0",
-    "pa11y-ci": "^3.1.0",
-    "pa11y-ci-reporter-html": "^7.0.0",
-    "pm2": "^5.4.2",
-    "prettier": "^3.3.3",
-    "sass": "^1.80.5",
-    "ts-jest": "^29.2.5",
-    "ts-node": "^10.9.2",
-    "tsx": "^4.19.1",
-    "typescript": "^5.5.4",
-    "wait-on": "^8.0.0",
-    "whatwg-fetch": "^3.6.20"
-  },
-  "overrides": {
-    "pa11y-ci": {
-      "pa11y": "^8.0.0"
+    "name": "nhs-notify-web-template-management",
+    "workspaces": [
+        "lambdas/authorizer",
+        "lambdas/backend-api",
+        "lambdas/backend-client",
+        "frontend",
+        "frontend/amplify",
+        "tests/test-team",
+        "utils"
+    ],
+    "scripts": {
+        "build": "npm run build --workspace frontend",
+        "copy-files-for-testing": "mkdir -p frontend/public/testing && cp ./frontend/amplify/functions/send-email/email-template.html ./frontend/public/testing/email-template.html",
+        "create-backend-sandbox": "./scripts/create_backend_sandbox.sh",
+        "destroy-backend-sandbox": "./scripts/destroy_backend_sandbox.sh",
+        "start": "npm run start --workspace frontend",
+        "test:accessibility": "npm run copy-files-for-testing && pa11y-ci -c ./tests/accessibility/.pa11y-ci.js"
```

Comment:
can we add these scripts

- typecheck
- fix:lint
- test:unit

To this file please. Saves me from having to run `npm run test:unit --workspaces`.

Reply:
and the dev command to run the web-ui, please.